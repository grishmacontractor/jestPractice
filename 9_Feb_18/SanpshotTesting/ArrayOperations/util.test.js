//import util from './util.js'
//import deepFreeze from 'deep-freeze'
util = require('./util.js')
deepFreeze = require('deep-freeze')
test("Add Item to a list", function(){
   let jslibs = ['React', 'Mobx']
    deepFreeze(jslibs)
   let result = util.addItemToList(jslibs,"Redux")
   //let expected = ['React', 'Mobx', 'Jest']
   expect(result).toMatchSnapshot()
});

test("Remove Item frm a list", function(){
    let jslibs = ['React', 'Mobx']
   deepFreeze(jslibs)
    let result = util.removeItemToList(jslibs,1)
    //let expected = ['React']
    expect(result).toMatchSnapshot()
});

test("Update Item frm a list", function(){
    let jslibs = ['React', 'Mobx']
    deepFreeze(jslibs)
    let result = util.updateItemFromList(jslibs,1,'Redux')
    //let expected = ['React','Redux']
    expect(result).toMatchSnapshot()
});