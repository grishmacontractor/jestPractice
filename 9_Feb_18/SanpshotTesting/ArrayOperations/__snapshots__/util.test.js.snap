// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Item to a list 1`] = `
Array [
  "React",
  "Mobx",
  "Redux",
]
`;

exports[`Remove Item frm a list 1`] = `
Array [
  "React",
]
`;

exports[`Update Item frm a list 1`] = `
Array [
  "React",
  "Redux",
]
`;
